lms/events/event.py

```diff
@@ -15,12 +15,12 @@ class BaseEvent:  # pylint:disable=too-many-instance-attributes
     Type = EventType.Type
```

Comment:
Unrelated to this PR, it would be helpful if the documentation said a bit more about the meaning of "event" in the context of this package. I gather it means "something interesting happened" for the purposes of tracking usage, auditing etc?